dominate/dom_tag.py

```diff
@@ -332,7 +332,9 @@ def _render(self, sb, indent_level, indent_str, pretty, xhtml):
     sb.append('<')
     sb.append(name)
 
-    for attribute, value in sorted(self.attributes.items()):
```

Comment:
No need for the new static method, just simply put a `if value is False: continue` into the loop body